### PR TITLE
locale updates for OJS 2.4.8-1

### PR DESCRIPTION
**Pull request by mtub**
Locale updates received from José Castro Silva

also included a tiny de_DE update in this pull request...

(https://pkp.sfu.ca/wiki/index.php?title=OJS:_Portuguese_(pt_PT))

pkp/pkp-lib#1560

### CODE REVIEW

**Comment by mtub** [comment]
@asmecher , I can take care of cherry-picking this to `ojs-stable-2_4_8`

File: help/pt_PT/intro/topic/000001.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
 	</section>
 	<section title='4. Por que aparece a mensagem "Esta revista não aceita submissões neste momento"?'>
 		<![CDATA[
-<p>Caso o Editor-Gestor ainda não tenha criado nenhuma secção para a revista (por exemplo, Artigos, Livros, Revisões, etc.), os autores verão esta mensagem durante o processo de submissão. Também, caso o Editor-Gestor tenha activado a opção "Restrita a Editores e Editores de Secção", para todas as secções criadas, somente os Editores e Editores de Secção poderão submeter à revista.</p>
```
Comment (asmecher) [inline help/pt_PT/intro/topic/000001.xml]: I'm uncertain about some of the role language changes here -- the pt_PT translation appears to refer to Journal Managers as "Editor-Gestor" (see e.g. `user.role.manager` in `locale/pt_PT/locale.xml`) but this change uses Director instead.

Comment (mtub) [inline help/pt_PT/intro/topic/000001.xml]: This is a change introduced by this update in all files, including the workflow images. I just discovered one missed line in admin.xml, but other than that this is very thorough.

Comment (asmecher) [inline help/pt_PT/intro/topic/000001.xml]: Ah, OK, thanks for checking.

Comment (asmecher) [inline help/pt_PT/intro/topic/000001.xml]: This is probably a subject different translators will fight over, but as long as we're consistent, I don't mind merging this. But let me spot-review it a little more first.

File: plugins/themes/custom/locale/pt_PT/locale.xml [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
+	<message key="plugins.theme.custom.notWritable">NOTA: A folha de estilos deste plugin, "{$stylesheetFileLocation}", não possui permissão de escrita. Este ficheiro deve possuir permissão de escrita antes usar o plugin.</message>
 	<message key="plugins.theme.custom.settings">Configurações</message>
 	<message key="plugins.theme.custom.pickColour">Escolha a cor</message>
 	<message key="plugins.theme.custom.header">Cabeçalho</message>
 	<message key="plugins.theme.custom.link">Link</message>
 	<message key="plugins.theme.custom.background">Cor de segundo plano</message>
 	<message key="plugins.theme.custom.foreground">Cor de primeiro plano</message>
-	<message key="plugins.theme.custom.notWritablePlugin">NOTA: A opção da stylesheet base da revista é activada automaticamente porque o ficheiro de estilos deste plugin, "{$ stylesheetFileLocation}" não tem permissões de escrita.</message>
+	<message key="plugins.theme.custom.notWritablePlugin">NOTA: A opção da stylesheet base da revista é activada automaticamente porque o ficheiro de estilos deste plugin, "{$stylesheetFileLocation}" não tem permissões de escrita.</message>
```
Comment (asmecher) [inline plugins/themes/custom/locale/pt_PT/locale.xml]: Good catch.

File: plugins/generic/pln/locale/de_DE/locale.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
 	<message key="plugins.generic.pln.settings.acron_required">Das PKP-PLN-Plug-In benötigt entweder das Acron-Plug-In oder ein Tool für regelmäßig wiederholte Aktionen (wie 'cron').</message>
 	<message key="plugins.generic.pln.settings.saved">PKP-PLN-Einstellungen gespeichert.</message>
 	<message key="plugins.generic.pln.settings.terms_of_use">Nutzungsbedingungen</message>
-	<message key="plugins.generic.pln.settings.terms_of_use_help">Als Hauptkontakt der Zeitschrift akzeptiere und bestätige ich die folgenden Nutzungsbedingungen zur Teilnahme am Private LOCKSS Network des Public Knowledge Projects (PKP-PLN):</message>
+	<message key="plugins.generic.pln.settings.terms_of_use_help">Als Zeitschriftenverwalter/in der Zeitschrift akzeptiere und bestätige ich die folgenden Nutzungsbedingungen zur Teilnahme am Private LOCKSS Network des Public Knowledge Projects (PKP-PLN):</message>
```
Comment (asmecher) [inline plugins/generic/pln/locale/de_DE/locale.xml]: Looks like some German snuck in here.

File: plugins/generic/objectsForReview/locale/pt_PT/locale.xml [hunk 4]
```diff
@@ -8,11 +8,550 @@
...
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.create">Criar campo metadados</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.edit">Editar campo metadados</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.confirmDelete">Esta acção irá eliminar todos os metadados inseridos neste campo. Tem a certeza de que deseja apagar o campo de metadados para esta tipo de revisão tipo do objecto?</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.copy">Copiar</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.name">Nome</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.required">Obrigatório</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.display">Mostrar</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.type.chooseType">Seleccionar tipo de campo metadados</message>
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata.type.smalltextfield">Single word text box</message>
```
Comment (asmecher) [inline plugins/generic/objectsForReview/locale/pt_PT/locale.xml]: Some English snuck in...

File: help/pt_PT/intro/topic/000001.xml (hunk 1 above)
Comment (mtub) [inline help/pt_PT/intro/topic/000001.xml]: Yes, but I am okay with this change. I wouldn't change a role's name in every occasion, but this seems to make sense and comes from our translator.

**Comment by asmecher** [comment]
Wow, that is a lot of work (even the diagrams are updated). Pass along big thanks to the translator. Just a couple of tweaks necessary and then it's ready to merge.

File: plugins/generic/pln/locale/de_DE/locale.xml (hunk 3 above)
Comment (mtub) [inline plugins/generic/pln/locale/de_DE/locale.xml]: see PR note: this is a multi-language update :)

File: plugins/generic/objectsForReview/locale/pt_PT/locale.xml (hunk 4 above)
Comment (mtub) [inline plugins/generic/objectsForReview/locale/pt_PT/locale.xml]: will drop it

File: plugins/generic/pln/locale/de_DE/locale.xml (hunk 3 above)
Comment (asmecher) [inline plugins/generic/pln/locale/de_DE/locale.xml]: Yep, spotted that too late!

**Comment by mtub** [comment]
Yes, this is an amazing update. :+1:

**Comment by asmecher** [comment]
Excellent -- merged! I'll leave you the cherry-pick.

**Comment by mtub** [comment]
pkp-lib changes coming in, too...

**Pull request merged**